packages/lesswrong/lib/ultraFeed/UltraFeedRepo.ts

```diff
@@ -0,0 +1,413 @@
+import { FeedItemSourceType, FeedSpotlight, PreDisplayFeedComment, PreDisplayFeedCommentThread, FeedFullPost, FeedPostMetaInfo, FeedCommentsThread, FeedCommentFromDb } from '../../components/ultraFeed/ultraFeedTypes';
+import Comments from '../../server/collections/comments/collection';
+import AbstractRepo from '../../server/repos/AbstractRepo';
+import { recordPerfMetrics } from '../../server/repos/perfMetricWrapper';
+import { prioritizeThreads, prepareCommentThreadForResolver  } from './ultraFeedThreadHelpers';
+import { recombeeApi, recombeeRequestHelpers } from '../../server/recombee/client';
+import { HybridRecombeeConfiguration } from '../../lib/collections/users/recommendationSettings';
+import { aboutPostIdSetting } from '@/lib/instanceSettings';
+
+const COMMENT_LOOKBACK_WINDOW = '90 days';
+
+export interface ThreadStatistics {
+  commentCount: number;
+  maxKarma: number;
+  sumKarma: number;
+  sumKarmaSquared: number;
+  averageKarma: number;
+  averageTop3Comments: number;
+}
+
+class UltraFeedRepo extends AbstractRepo<"Comments"> {
```

Comment:
Why is this extending `AbstractRepo<"Comments">` rather than `AbstractRepo<"UltraFeedEvents">?